Updated Pull Logic for Parts 1 and 3, moved common into a sub-package…

WhiteAu opened the pull request:
Updated Pull Logic for Parts 1 and 3, moved common into a sub-package, 1st pass moving DB state values to Global Vars

WhiteAu committed:
Updated Pull Logic for Parts 1 and 3, moved common into a sub-package, 1st pass moving DB state values to Global Vars

WhiteAu committed:
style update

WhiteAu committed:
style updates and creating resynth job note w/ 'running' status

OptimusRhine reviewed (approved):
LGTM - we'll just want to make sure and verify each of these query changes once the code's testable in the pipeline.

A few optional suggestions below.

ddong63 reviewed (commented):
Generally looks good to me. Have few comments/questions below.

WhiteAu committed:
refactored Global String-Status Types to common

This also changes Batch/Note level Running/Success/Failure to use the same status strings. This changes Note Success to be indicated by 'completed' going forward.

WhiteAu committed:
Further Updates to Scheduling Logic in Parts 1 and 3

OptimusRhine reviewed (approved):
LGTM 👍
